monitor/index.ts

```diff
@@ -1,30 +1,80 @@
 import { SDK } from "codechain-sdk";
-import { EmailClient } from "./EmailNotify";
+import * as chainErrors from "./Alert";
+import { CodeChainAlert } from "./Alert";
 import { SlackNotification } from "./SlackNotify";
-import { getConfig } from "./util";
-
-async function main() {
-  const rpcUrl = getConfig<string>("rpc_url");
-  const networkId = getConfig<string>("network_id");
+import { getConfig, unsetBitIndices } from "./util";
 
-  const sdk = new SDK({ server: rpcUrl, networkId });
-  const emailClient = new EmailClient("");
+const targetEmail = "devop@kodebox.io";
```

Comment:
Please make the email destination configurable.

Reply:
Now the target email address is configurable.